app/controllers/releases_controller.rb

```diff
@@ -0,0 +1,76 @@
+class ReleasesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_release, only: [:edit, :update, :show, :destroy]
+
+  def index
+    @releases = Release.all.order(created_at: :desc)
+  end
+
+  def new
+    @release = Release.new
+  end
+
+  def edit
+  end
+
+  def show
+  end
+
+  def create
+    # create Method with Error Handling:
+    # If @release.save return non-nil values, that means they succeeded,
+    # and we can redirect back to the release page with a success message.
+    # If they return nil, we have the else processing:
+    # First collect the error and put it in the flash.now.alert
+    # Then do the render to :new.
```

Comment:
Please remove these comments. Comments should be used to help people understand complex code, what it is doing isn't obvious or if it calls methods that are located elsewhere.